CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 * Add support for Gherkin's [Rule/Example syntax](https://cucumber.io/docs/gherkin/reference/#rule)
 * Add `transpose` method to [data table interface](docs/support_files/data_table_interface.md)
 * Add `log` function to world, providing a shorthand to log plain text as [attachment(s)](docs/support_files/attachments.md)
+* `cucumber-js` is now written in [TypeScript](https://www.typescriptlang.org/) and includes its own typings, so you no longer need to depend on `@types/cucumber` in your own TypeScript projects
```

Comment:
I think this could be shortened to:

> Now includes `Typescript` type definitions, deprecating the need for `@types/cucumber` in TypeScript projects

Thoughts?

Reply:
Agree that's better, updated.

I had a look at the tests from DefinitelyTyped and ran them with tsd, there are quite a few inaccuracies (e.g. things like `Around` and `defineSupportCode` that we don't export, plus some naming differences of classes and interfaces. Not sure if you think we need to call it out in the changelog, but I think TypeScript users have probably come to expect this when a library starts publishing its own types.